docs/pages/api/badge.js

```diff
@@ -5,7 +5,14 @@ import MarkdownDocs from 'docs/src/modules/components/MarkdownDocs';
 import markdown from './badge.md';
 
 function Page() {
-  return <MarkdownDocs markdown={markdown} />;
+  return (
+    <MarkdownDocs
+      HeadProps={{
+        children: <meta name="robots" content="noindex,follow" />,
```

Comment:
So if I get this right we tell crawlers not to index them? When I search for `table` or `snackbar` I never get the link to the API page in the first page anyway.

What problem is this solving?

Reply:
Ok only in google. For duckduckgo I get for `table`:
- 

and `snackbar`:
- https://material-ui.com/components/snackbars/
- https://material-ui.com/api/snackbar/
- https://material.io/design/components/snackbars.html

which is IMO a perfect order of results. Examples for starters, API for in-depth, spec for mockups

Reply:
I wanted to avoid keyword cannibalization. Reading https://www.semrush.com/blog/keyword-cannibalization-harming-on-site-seo/, it seems to be fine. Reverting.
> When Cannibalization Doesn’t Matter